src/main/resources/insurance_customer_activity.avro

```diff
@@ -0,0 +1,116 @@
+{
+    "namespace": "insurance",
+    "name": "insurance_customer_activity",
+    "type": "record",
+    "fields": [
+        {
+            "name": "activity_id",
+            "type": {
+                "type": "int",
+                "arg.properties": {
+                    "iteration": {
+                        "start": 1,
+                        "step": 1
+                    }
+                }
+            }
+        },
+        {
+            "name": "customer_id",
+            "type": {
+                "type": "int",
+                "arg.properties": {
+                    "range": {
+                        "min": 1,
+                        "max": 1000
+                    }
+                }
+            }
+        },
```

Comment:
should line up with `insurance_customers` i.e. have the same values so that we don't emit activity that doesn't have corresponding customer record